src/components/Todos/TodoItem.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  return (
+    <div
```

Comment:
if you don't have any logic in component(state, methods, consts, etc) you don't need a return key word
```suggestion
export const TodoItem: React.FC<Props> = ({ todo }) => (
    <div
```

Reply:
I made all the above corrections in the final part of the task. Made a separate commit for convenience

https://github.com/mate-academy/react_todo-app-with-api/pull/104/commits/7dbcfa958fb6aadb673713b6548f109c319679c2

Reply:
ok 👍